simpleclient/src/main/java/io/prometheus/client/Histogram.java

```diff
@@ -247,6 +254,20 @@ public Timer startTimer() {
     return noLabelsChild.startTimer();
   }
 
+  /**
+   * Executes runnable code (ie Java 8 Lambda) and observes a duration of how long it took to run.
+   *
+   * @param timeable Code that is being timed
+   */
+  public void time(Runnable timeable){
```

Comment:
We had requests for the existing timers to return the duration for use in logging. These should probably do the same.